packages/ember-htmlbars/lib/helpers/each.js

```diff
@@ -11,7 +11,7 @@ import decodeEachKey from 'ember-htmlbars/utils/decode-each-key';
   of the base Handlebars `{{#each}}` helper.
 
   The default behavior of `{{#each}}` is to yield its inner block once for every
-  item in an array.
+  item in an array passing the item as the first param to the block.
```

Comment:
We try to refer to `param` as `parameter` in prose for the API docs. "first block parameter" is fine.